packages/internal/rollup-plugin-bundle-manifest/src/index.ts

```diff
@@ -36,7 +36,7 @@ export default function bundleManifest({
 } = {}): Plugin {
   return {
     name: 'bundle-manifest',
-    generateBundle(outputOptions, bundle) {
+    generateBundle(outupsertOptions, bundle) {
```

Comment:
Wondered if this may not have been intentional.

Reply:
lol I think i did a find and replace which messed this up